packages/renderers-rust/e2e/memo/Cargo.toml

```diff
@@ -11,7 +11,8 @@ test-sbf = []
 
 [dependencies]
 anchor-lang = { version = "0.30.0", optional = true }
-borsh = "^0.10"
+borsh = "^1.5.3"
```

Comment:
Same comment here about the `borsh` version.

Reply:
switching to borsh 0.10